### ISSUE TITLE
log out while I'm playing game

### ISSUE THREAD

**Issue by kismp123** [mayankk2308/purge-wrangler#20]
Hi, I use this script very useful.
First, I wanna appreciate you guys👍 

While I use egpu on MacBook Pro 2015 mid with this script, I found a bug.
It makes my MacBook log out in some situation.
For example, while I'm encoding a video with Final Cut pro, my monitor(I use a monitor with) has broken. Then my MacBook log out automatically.
And It's occurred while playing some games
I found a temporary solution which is enabling display mirroring.

Basically I don't enable display mirroring because It makes my Mac slower.

Thank you

**Comment by mayankk2308** [comment]
Hi, please follow the **issue template** for submitting issues. What Mac model is this (13 vs. 15), what is the eGPU configuration, and the version of macOS?

**Comment by mayankk2308** [comment]
Hi, please provide an update with respect to the issue if possible. The current information is insufficient to determine the problem. Issue will be closed soon as it is not script/patch related but more GPU/Framebuffer related, and if no response is provided.

**Comment by kismp123** [comment]
I have 15 inch 2015 mid and 10.13.6 high Sierra for Mac

**Comment by mayankk2308** [comment]
Does your Mac have an AMD discrete GPU? Are you using an NVIDIA eGPU? If yes to both, there is a workaround mentioned in the README (for slow rendering/black screens which can be followed, even for this problem. Or you can do `sudo pmset -a gpuswitch 0` for every boot. This issue has previously come up on the eGPU forums in different forms (black screen/stuttering). If the framebuffer crashes, the system logs out. No other workarounds are possible and the issue is not with the script/patch itself, but the configuration.

If you answered no to any of the above questions, then it is very difficult to determine the problem. It may be hardware related - cannot pinpoint. The only way would be to test the eGPU as a whole with another system, or test on a clean macOS install. In either case, the patch/script seem to be working as expected. I also recommend posting on eGPU.io forums for more visibility.
